.travis.yml

```
@@ -34,7 +34,7 @@ before_install:
 script:
   - export PYTHON=""  # Configure PyCall to use the Conda.jl package's Python
   - OPTIONS=$(julia -e 'VERSION == v"0.6.0" && print("--inline=no")')  # https://github.com/JuliaLang/julia/issues/22582
-  - julia $OPTIONS -e 'Pkg.clone(pwd()); Pkg.build("SyntheticGrids"); Pkg.checkout("PyCall"); Pkg.test("SyntheticGrids"; coverage=true)'
+  - julia $OPTIONS -e 'Pkg.clone(pwd()); Pkg.build("SyntheticGrids"); ENV["PYTHON"] = ""; Pkg.build("PyCall"); Pkg.test("SyntheticGrids"; coverage=true)'
```

```suggestion
  - julia $OPTIONS -e 'Pkg.clone(pwd()); Pkg.build("SyntheticGrids"); Pkg.build("PyCall"); Pkg.test("SyntheticGrids"; coverage=true)'
```
Does this work? The `export` line above should already be setting that variable

---

Looks like that was necessary, even though it was kind of a shot in the dark. As of https://github.com/invenia/SyntheticGrids.jl/pull/16/commits/d29636da0bb3cdbee16f55bf4ee1b04ef75a8145 it was breaking with 
```julia 
INFO: Testing SyntheticGrids
ERROR: LoadError: LoadError: could not load library "/home/travis/.julia/v0.6/Conda/deps/usr/lib/libpython3.7m.so.1.0"
```
https://travis-ci.org/invenia/SyntheticGrids.jl/jobs/538394089

---

But that commit didn't have `Pkg.build("PyCall")`